.env

```diff
@@ -0,0 +1 @@
+REACT_APP_BASE_URL=http://localhost:3000
```

Comment:
Regarding .env: usually we gitignore it, and create .env.example file with sample values